core/src/providers/mistral.rs

```diff
@@ -104,13 +105,16 @@ struct MistralChatMessage {
     pub name: Option<String>,
     #[serde(skip_serializing_if = "Option::is_none")]
     pub tool_calls: Option<Vec<MistralToolCall>>,
+    #[serde(skip_serializing_if = "Option::is_none")]
+    pub tool_call_id: Option<String>,
```

Comment:
Why can it be None ? If none, we'll need a default no ? IIRC, mistral API requires it

Reply:
They want it only if tool message? and for other they don't want to hear about it?

Reply:
Yes nvm me I didn't understand that this was used for all kinds of msg originally 👍